src/main.ts

```diff
@@ -59,6 +63,9 @@ export async function run() {
       auth.configAuthentication(registryUrl, alwaysAuth);
     }
 
+    const corepack = core.getInput('corepack') || 'false';
```

Comment:
given that the yaml is structured and supports array, why not specifying the `corepack` input in an array format?

Reply:
For simplicity, i would prefer to keep it using a string as the input. But probably we can also make it support array, however currently there's no built-in function to do that.